examples/backends/sglang/launch/xpu/disagg_pd_xpu.sh

```
@@ -0,0 +1,523 @@
+################################################################################
+
+stop_services() {
+    log_info "Stopping all services..."
+
+    # Kill all SGLang and router processes
+    pkill -9 -f "sglang.launch_server" 2>/dev/null || true
+    pkill -9 -f "sglang_router.launch_router" 2>/dev/null || true
+
+    # Free ports
+    for port in $ROUTER_PORT $PREFILL_PORT $DECODE_PORT $BOOTSTRAP_PORT; do
+        fuser -k ${port}/tcp 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)